backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -3390,12 +3004,12 @@ template <typename Torus> struct int_scalar_mul_buffer {
     }
   }
 
-  void release(cudaStream_t *streams, uint32_t *gpu_indexes,
+  void release(cudaStream_t const *streams, uint32_t const *gpu_indexes,
```

Comment:
is const properly enforced by C++ compilers ? I don't recall

Reply:
Yes, but as always with c++ there are ways around it I guess.

Reply:
most likely but if you don't "cheat" it should be so that's good :)